Timezone issue

This is related to issue #68, but the problem seems to be broader, so I'm posting this separately.

_sheetstorm/utils/base_tests.py_, line **122** returns an error, such as **_AssertionError: FakeDatetime(2019, 3, 6, 14, 41, 33, 947105, tzinfo=<UTC>) != FakeDatetime(2019, 3, 6, 14, 41, 33, 947105)_** **ONLY IF** we have a TIME_ZONE configured in local settings (as _Europe/Warsaw_).
- The first value is stored in the database, in an **automated DateTimeField**.
- The second value is a result of calling **datetime.datetime.now()**

I understand that the issue is lack of timezone info in **datetime.now()** result. But for instance,  **.timestamp()** presented in **float** gives a **1 hour** offset between the two values.

Okay, so the **datetime.now()** must be returning time in local timezone setting, right?

Okay, now we're getting to the good part: we have a **.now()** method and a **.utcnow()** method to specifically return UTC time. And guess what? Here, both **datetime.now()** and **datetime.utcnow()** will return **THE SAME VALUE!**

BUT, if I use **.astimezone()** and I convert it to the UTC, **datetime.astimezone(timezone.utc)** NOW it returns an hour earlier!

```
ipdb> datetime.datetime.now()                                                   
FakeDatetime(2019, 3, 6, 14, 42, 20, 717704)
ipdb> datetime.datetime.utcnow()                                                
FakeDatetime(2019, 3, 6, 14, 42, 20, 717704)
ipdb> datetime.datetime.now().astimezone(datetime.timezone.utc)                 
FakeDatetime(2019, 3, 6, 13, 42, 20, 717704, tzinfo=datetime.timezone.utc)
```

And it seems no matter what combination I try, if I convert both compared values to same timezone there will ALWAYS be this offset!

Does anyone have any idea what on earth is going on here? To me it seems like Python itself dropping a ball here. I have no idea how to fix this thing, except for maybe not configuring the local timezone at all.

I found [this thread](https://stackoverflow.com/questions/20804837/weird-astimezone-behavior). Maybe it's related with this issue?

--- a/utils/base_tests.py
+++ b/utils/base_tests.py
@@ -1,7 +1,7 @@
-import datetime
 from django.core.exceptions import ValidationError
-from django.test import TestCase
 from django.db.models.base import Model
+from django.test import TestCase
+from django.utils import timezone
 from freezegun import freeze_time
 from rest_framework.serializers import BaseSerializer
 
@@ -109,7 +109,7 @@ class BaseModelTestCase(TestCase):
     Using any other time-related type can result in fail!
     """
     def field_auto_now_test(self, auto_field, update_field, value):
-        with freeze_time(datetime.datetime.now()) as frozen_datetime:
+        with freeze_time(timezone.now()) as frozen_datetime:
             model = self.default_model()
             model.full_clean()
             model.save()
@@ -118,7 +118,7 @@ class BaseModelTestCase(TestCase):
             model.full_clean()
             model.save()
             current_value = getattr(model, auto_field)
-            self.assertEqual(current_value.timestamp(), datetime.datetime.now().timestamp())
+            self.assertEqual(current_value, timezone.now())
 
     """
     Test that specified field should be given default non-null value,

